src/frontends/tensorflow/src/pass/transpose_sinking.cpp

```diff
@@ -88,7 +88,7 @@ static shared_ptr<Transpose> combine_transposes(const shared_ptr<Transpose>& t1,
     auto perm_t1 = apply_permutation(default_order, t1_const->get_axis_vector_val());
     auto perm_t2 = apply_permutation(perm_t1, t2_const->get_axis_vector_val());
 
-    auto combined = make_transpose(t2->input_value(0), perm_t2);
+    auto combined = make_transpose(t1->input_value(0), perm_t2);
```

Comment:
@evkotov, please tell - what is the history that you take a look at this code?:)
Could you please elaborate a bit more what is wrong with it code because the impact can be huge?

Reply:
Please take look at the test in this PR
![image](https://user-images.githubusercontent.com/21181928/185331598-ead30928-6fdb-4997-96c1-f6ac5adb5831.png)
Previously we used 2nd Transpose op here which is wrong :)